SEDaily-IOS/HeaderView.swift

```diff
@@ -180,8 +184,11 @@ extension HeaderView {
     }
     
     @objc func downVoteButtonPressed() {
-        guard User.checkAndAlert() else { return }
-        guard let podcastId = model.podcastId else { return }
+        guard UserManager.sharedInstance.isCurrentUserLoggedIn() == true else {
+            Helpers.alertWithMessage(title: Helpers.Alerts.error, message: Helpers.Messages.youMustLogin, completionHandler: nil)
+            return
+        }
+        let podcastId = model._id
         API.sharedInstance.downvotePodcast(podcastId: podcastId, completion: { (success, active) in
             guard success != nil else { return }
             if success == true {
```

Comment:
Is it necessary to have the switch statement? Could you not pass active directly into subtractScore?

Reply:
This does not have to be a switch statement. I have no idea the past code the led to this. Fixed